packages/workbox-cli/src/index.d.ts

```diff
@@ -0,0 +1,12 @@
+declare module 'workbox-build' {
```

Comment:
Hmmm, hopefully having a `d.ts` file in the `src` directory won't cause any issues. Could this also be at the top level?

Reply:
Could you be more specific about what issues it might cause?

FWIW, the output in `build/` looks like:

```
$ ls packages/workbox-cli/build/
app.d.ts        app.js          bin.d.ts        bin.js          lib
```

without the `index.d.ts` carried over. We only publish the `build/` directory to `npm`, so I don't think anything would "leak".

Reply:
(Moving it to the top-level meant that it wasn't automatically picked up by `tsc`. I guess I could change the compiler settings to override that, but unless there's an actual problem, I'd rather keep it simple.)

Reply:
...I'm just going to merge (as this is causing problems for me whenever I run `gulp build` locally) and if there's some reason that it needs to be moved, we can do that in a follow-up PR.